pallets/delegation/src/try_state.rs

```diff
@@ -19,61 +19,64 @@
 use frame_support::ensure;
 use kilt_support::test_utils::log_and_return_error_message;
 use scale_info::prelude::format;
+use sp_runtime::TryRuntimeError;
 
 use crate::{Config, DelegationHierarchies, DelegationNodeOf, DelegationNodes};
 
-pub(crate) fn do_try_state<T: Config>() -> Result<(), &'static str> {
-	DelegationNodes::<T>::iter().try_for_each(|(delegation_node_id, delegation_details)| -> Result<(), &'static str> {
-		let hierarchy_id = delegation_details.hierarchy_root_id;
+pub(crate) fn do_try_state<T: Config>() -> Result<(), TryRuntimeError> {
+	DelegationNodes::<T>::iter().try_for_each(
+		|(delegation_node_id, delegation_details)| -> Result<(), TryRuntimeError> {
+			let hierarchy_id = delegation_details.hierarchy_root_id;
```

Comment:
Maybe we could move this closure into it's own function. I think that would improve readability.

Reply:
Refactored in: [fa04854](https://github.com/KILTprotocol/kilt-node/pull/541/commits/fa048542f91dc3495838f1b57849113080852323)